spec/services/appeal_decision_tree/challenged_decision_status_spec.rb

```diff
@@ -17,7 +24,23 @@
     context 'and the status is other than `pending`' do
       let(:challenged_decision_status) { ChallengedDecisionStatus::RECEIVED }
 
-      it { is_expected.to have_destination(:dispute_type, :edit) }
+      context 'for a case with disputes' do
+        let(:case_type) { CaseType::BINGO_DUTY }
```

Comment:
I think we should use a dummy case type here, e.g.

```ruby
let(:case_type) { CaseType.new(ask_dispute_type?: true) }
```

... just in case the values for bingo duty change at some point!